src/components/Tooltip/BaseGenericTooltip/index.native.tsx

```diff
@@ -46,13 +47,12 @@ function BaseGenericTooltip({
     const rootWrapper = useRef<RNView>(null);
 
     const StyleUtils = useStyleUtils();
-
-    const {animationStyle, rootWrapperStyle, textStyle, pointerWrapperStyle, pointerStyle} = useMemo(
+    const {rootWrapperStyle, textStyle, pointerWrapperStyle, pointerStyle} = useMemo(
         () =>
             StyleUtils.getTooltipStyles({
                 // eslint-disable-next-line react-compiler/react-compiler
                 tooltip: rootWrapper.current,
-                currentSize: animation,
+                currentSize: animation.value,
```

Comment:
This line should be removed, right? And the logic inside getTooltipStyles adjusted so that animationStyles aren't returned.

Reply:
I will correct it, I didn't want to change utils that are already written, but I can safely remove it is not used anywhere